client/src/components/v2/CameraRecording.tsx

```diff
@@ -0,0 +1,98 @@
+import { VideoFeedStatus } from 'api/v2/camera';
+import React from 'react';
+import { Badge, Button, Card, Col, Row } from 'react-bootstrap';
+import styles from './CameraRecording.module.css';
+import CameraRecordingStatus, { CameraRecordingStatusProps } from './CameraRecordingStatus';
+
+export type CameraRecordingPropT = VideoFeedStatus & CameraRecordingStatusProps & { ip: string };
+
+export interface CameraRecordingProps {
+  /** Information for primary device */
+  primary: CameraRecordingPropT
+  /** Information for secondary device */
+  secondary: CameraRecordingPropT
+  startRecording: () => void
+  stopRecording: () => void
+}
+
+/**
+ * Camera recording buttons for starting and stopping video recording.
+ *
+ * Starts/stops recording for both displays.
+ *
+ * This is a feature intended for V3 but is currently in V2 for testing.
```

Comment:
I'd argue it's intended for both! Both V2 and V3 camera systems will run the same code and can both record.

Reply:
Done
b4b461d